packages/react-dev-utils/FlowTypecheckPlugin.js

```diff
@@ -3,7 +3,7 @@ var path = require('path');
 var chalk = require('chalk');
 var childProcess = require('child_process');
 var flowBinPath = require('flow-bin');
-const flowTypedPath = path.join(__dirname, 'node_modules', '.bin', 'flow-typed');
+const flowTypedPath = path.join(__dirname, '..' , '..', 'node_modules', '.bin', 'flow-typed');
```

Comment:
This is brittle, can you use something like `require.resolve` instead to find `flow-typed` and then find a `.bin` from it?